### DISCUSSION TITLE
CLI: doctor warning on large chunks

### DISCUSSION THREAD

**Discussion by wkalt** [foxglove/mcap#1558] [General]
It would be useful if the `doctor` subcommand printed a warning on large chunks. Going from memory, I think we have used 4MB, 8MB, and 10MB as chunk sizes in various contexts. The warning should be well above those numbers since single oversized messages can create oversized chunks. 

50MB would probably be a sufficiently large threshold to trigger primarily in cases where mis-configuration is to blame.

**Comment by james-rms** [comment]
People can have much larger messages than 50MB, i think it makes sense to only warn if the chunk size gets to some size too big to allocate a single buffer for on certain platforms, say `2**30`.
